src/main/webapp/WEB-INF/jpa-configuration.xml

```diff
@@ -50,11 +50,16 @@
         </property>
     </bean>
 
-    <bean id="dataSource" class="org.springframework.jdbc.datasource.DriverManagerDataSource">
-        <property name="driverClassName" value="${org.osiam.resource-server.db.driver}"/>
-        <property name="url" value="${org.osiam.resource-server.db.url}"/>
-        <property name="username" value="${org.osiam.resource-server.db.username}"/>
-        <property name="password" value="${org.osiam.resource-server.db.password}"/>
+    <bean id="hikariConfig" class="com.zaxxer.hikari.HikariConfig">
+        <property name="poolName" value="springHikariCP" />
```

Comment:
I'm not sure if the pool name is logged or used somewhere (maybe later with metrics?), but using the name of an hikari spring example isn't that helpful. My suggestion: `osiam-resource-server-connection-pool`